google/cloud/pubsub_v1/subscriber/_protocol/dispatcher.py

```diff
@@ -66,6 +67,13 @@
 IDs at a time.
 """
 
+_MIN_ACK_MODACK_RETRY_DURATION_SECS = 1
+"""The time to wait for the first retry of failed acks and modacks when exactly-once
+is enabled."""
+
+_MAX_ACK_MODACK_RETRY_DURATION_SECS = 20 * 60
```

Comment:
This is 20 mins. The max ack deadline is 10 mins.  What's the reason to make this 20?

Reply:
It's a just-in-case thing. I will change it to 10min since there's no stronger reason than that.